pkg/common/configs/config_test.go

```diff
@@ -659,6 +659,8 @@ partitions:
 	stateDumpFilePath := "yunikorn-state.txt"
 	assert.Equal(t, conf.Partitions[0].StateDumpFilePath, stateDumpFilePath)
 	assert.Equal(t, conf.Partitions[1].StateDumpFilePath, "")
+	err = os.Remove(stateDumpFilePath)
```

Comment:
This should be in a `defer` call. 
Also, just use `assert.NilError()`.

Reply:
done.